docs/api_reference/python-api-reference.md

```diff
@@ -19,6 +19,11 @@ SPDX-License-Identifier: MPL-2.0
 .. automodule:: power_grid_model.enum
    :undoc-members:
    :show-inheritance:
+
+.. automodule:: power_grid_model.typing
+   :members: ComponentAttributeFilterOptions, ComponentAttributeMapping
```

Comment:
Hmm, interesting. I'll take another look.

Reply:
also, it's confusing to our users that there's a section called `enum` but this object lives in `typing`

Reply:
I wouldn't mind displace it if you don't mind ;)

Reply:
How does it look now?

Reply:
`ComponentAttributeFilterOptions` is gone now. please re-add.

`ComponentAttributeMapping` makes sense as is, but i'm still thinking whether it's good to add a separate `## typing` subsection because that's what the `import` module is called

Reply:
Are you sure? https://power-grid-model--758.org.readthedocs.build/en/758/api_reference/python-api-reference.html#power_grid_model.enum.ComponentAttributeFilterOptions

Reply:
oh weird... it disappeared on me, but now it's back :+1: apologies